### PR TITLE
WD-16647 Redesigned internet-of-things/smart-home

### PR DESCRIPTION
**Pull request by muhammad-ali-pk**
## Done

- Redesigned ubuntu.com/internet-of-things/smart-home

## QA
- View the site in your web browser at: https://ubuntu-com-14530.demos.haus/internet-of-things/smart-home
    - Be sure to test on mobile, tablet and desktop screen sizes
- Match the website with Design and Copydoc.
  - [Design](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=2001-16452&t=4kUhsH39i1i4UsV9-1)
  - [Copydoc](https://docs.google.com/document/d/1zwB-6Flt0Jm7XIxhWUWRJjKC1WCGVmVVNmPHtyOTHuE)

## Issue / Card

Fixes #[16647](https://warthogs.atlassian.net/browse/WD-16647)

### CODE REVIEW

**Comment by mattea-turic** [comment]
thanks @muhammad-ali-pk :) only two things:

For this section:
Screenshot 2024-12-03 at 11 53 42
- [ ] the asset I uploaded seems to have rendered without the blend mode being applied, so I've re-exported [here](https://assets.ubuntu.com/v1/736618a0-hero-img.png) for you to use, with the bg already applied so dw about adding a transparency layer for this one

For this section:
Screenshot 2024-12-03 at 12 07 30
- [ ] could you try removing the wrapping from the image? there looks to be too much space between the container and the h5, esp. on smaller screens

thanks in advance!

**Comment by muhammad-ali-pk** [comment]
Thanks @mattea-turic ! Done :)

**Comment by eliman11** [comment]
Looks great thanks @muhammad-ali-pk! Just one comment from me :) 
- [ ] Without changing styling, could you change the h5 tags in these 3 sections into h2?
Screenshot 2024-12-03 at 12 53 27
Screenshot 2024-12-03 at 12 53 16
Screenshot 2024-12-03 at 12 53 34

**Comment by muhammad-ali-pk** [comment]
Thanks @eliman11 ! Done :)

**Comment by mattea-turic** [comment]
Looks great! +1ing

**Comment by eliman11** [comment]
Thanks! Adding +1

File: templates/internet-of-things/smart-home.html [hunk 1]
```diff
@@ -10,210 +10,268 @@
...
+        critical. Canonical can provide the support and security maintenance you need so your team can focus on developing
+        world-class products.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="#get-in-touch" class="p-button--positive">Contact us to get started</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--cinematic is-cover is-highlighted">
+        <img class="p-image-container__image"
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/internet-of-things/smart-home.html:39]: I will supplement this comment as I see you have actively removed the use of the image template. If this is for a specific reason you can leave it as it. I would however reach out to @immortalcodes as he might have some suggestions for better use

File: templates/internet-of-things/smart-home.html [hunk 2]
```diff
@@ -10,210 +10,268 @@
...
-        }}
+      <div class="col">
+        <div class="p-image-container is-cover is-highlighted">
+          <img class="p-image-container__image"
+               src="https://assets.ubuntu.com/v1/c8af9315-universal-compatibility.png"
+               alt="Universal Compatibility Image" />
+        </div>
+        <h3 class="p-heading--5">
+          Devices that speak a common language
+        </h3 class="p-heading--5">
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:107]:
```suggestion
        </h3>
```

File: templates/internet-of-things/smart-home.html [hunk 3]
```diff
@@ -10,210 +10,268 @@
...
+        announced</a> its membership in the Connectivity Standards Alliance and its upcoming support for the Matter
+        standard. <a href="#get-in-touch" class="js-invoke-modal">Sign up</a> to receive smart home and IoT news.
       </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="u-fixed-width">
+  <section class="p-section">
+    <div class="row p-section--shallow">
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/internet-of-things/smart-home.html:116]: We try to separate structural and spacing elements like this

File: templates/internet-of-things/smart-home.html [hunk 4]
```diff
@@ -10,210 +10,268 @@
...
+            </div>
+          </div>
+        </div>
       </div>
     </div>
+  </section>
 
-    <div class="u-fixed-width">
+  <section class="p-section--deep">
+    <div class="row p-section--shallow">
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:190]:
```suggestion
    <div class="u-fixed-width">
      <div class="p-section--shallow">
```

File: templates/internet-of-things/smart-home.html (hunk 1 above)
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:39]:
```suggestion
        <img class="p-image-container__image u-hide--small"
```

**Comment by muhammad-ali-pk** [comment]
Thanks @petesfrench ! I have incorporated all your code suggestions in the new commit.
PS., awesome guide. Thanks for sharing :)

File: templates/internet-of-things/smart-home.html [hunk 5]
```diff
@@ -10,213 +10,296 @@
...
-          Over-the-air updates always have an element of risk. With <a href="/core">Ubuntu Core</a> you have the assurance that a failed update will never affect your device.
-        </p>
+      <div class="col">
+        <div class="p-image-container is-cover is-highlighted">
+          {{ image(url="https://assets.ubuntu.com/v1/c8af9315-universal-compatibility.png",
+                    alt="",
+                    width="1800",
+                    height="1015",
+                    hi_def=True,
+                    loading="auto|lazy",
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:114]:
```suggestion
                    loading="lazy",
```

File: templates/internet-of-things/smart-home.html [hunk 6]
```diff
@@ -10,213 +10,296 @@
...
+        <div class="p-equal-height-row">
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/166662b7-reliable-ota-updates.png",
+                                alt="",
+                                width="852",
+                                height="1279",
+                                hi_def=True,
+                                loading="auto|lazy",
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:144]:
```suggestion
                                loading="lazy",
```

File: templates/internet-of-things/smart-home.html [hunk 7]
```diff
@@ -10,213 +10,296 @@
...
+          </div>
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/a6fc8f4d-organize-your-fleet.png",
+                                alt="",
+                                width="852",
+                                height="1279",
+                                hi_def=True,
+                                loading="auto|lazy",
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:168]:
```suggestion
                                loading="lazy",
```

File: templates/internet-of-things/smart-home.html [hunk 8]
```diff
@@ -10,213 +10,296 @@
...
+          </div>
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/79509554-integrate-with-cloud-solutions.png",
+                                alt="",
+                                width="852",
+                                height="1279",
+                                hi_def=True,
+                                loading="auto|lazy",
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:193]:
```suggestion
                                loading="lazy",
```

File: templates/internet-of-things/smart-home.html [hunk 9]
```diff
@@ -10,213 +10,296 @@
...
-    <div class="u-fixed-width">
-      <hr class="p-rule" />
-    </div>
+                  </p>
+                  {{ image(url="https://assets.ubuntu.com/v1/2fcfbc3b-snap-store.png",
+                                    alt="",
+                                    width="1800",
+                                    height="1014",
+                                    hi_def=True,
+                                    loading="auto|lazy",
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:258]:
```suggestion
                                    loading="lazy",
```

File: templates/internet-of-things/smart-home.html [hunk 10]
```diff
@@ -10,213 +10,296 @@
...
+                    height="1015",
+                    hi_def=True,
+                    loading="auto|lazy",
+                    attrs={"class": "p-image-container__image"}) | safe
+          }}
+        </div>
+        <h3 class="p-heading--5">Devices that speak a common language</h3>
+        Canonical <a href="https://ubuntu.com/blog/canonical-joins-the-connectivity-standards-alliance">recently
+        announced</a> its membership in the Connectivity Standards Alliance and its upcoming support for the Matter
+        standard. <a href="#get-in-touch" class="js-invoke-modal">Sign up</a> to receive smart home and IoT news.
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:121]: The `js-invoke-modal` attaches `#get-in-touch` hash to users with JS enabled. We tend to set the fallback href to a static contact page for users who do not have JS enabled.

```suggestion
        standard. <a href="/internet-of-things/contact-us" class="js-invoke-modal">Sign up</a> to receive smart home and IoT news.
```

File: templates/internet-of-things/smart-home.html [hunk 11]
```diff
@@ -10,213 +10,296 @@
...
+                    alt="",
+                    width="1800",
+                    height="1015",
+                    hi_def=True,
+                    loading="auto|lazy",
+                    attrs={"class": "p-image-container__image"}) | safe
+          }}
+        </div>
+        <h3 class="p-heading--5">Devices that speak a common language</h3>
+        Canonical <a href="https://ubuntu.com/blog/canonical-joins-the-connectivity-standards-alliance">recently
```
Comment (britneywwc) [inline templates/internet-of-things/smart-home.html:119]:
```suggestion
        Canonical <a href="/blog/canonical-joins-the-connectivity-standards-alliance">recently
```

**Review by britneywwc — COMMENTED** [review]
Left a few comments! As for the image loading options, we tend to use `auto` when the images will be in the viewport on load (e.g hero) and lazy when not

**Review by britneywwc — APPROVED** [review]
thanks for making the changes, looks great!! +1

**Pull request merged**